app/build.gradle

```diff
@@ -18,6 +18,19 @@ android {
     dexOptions {
         javaMaxHeapSize "4g"
     }
+
+    packagingOptions {
```

Comment:
Good stuff - had to add this locally too, I think we looked at the same stackoverflow thread :)